app/assets/javascripts/discourse/app/controllers/topic.js

```diff
@@ -157,6 +158,25 @@ export default class TopicController extends Controller {
     return loaded && isSharedDraft;
   }
 
+  @discourseComputed("model.details.can_edit")
+  topicTitleClass(canEdit) {
+    if (canEdit) {
+      return "edit-topic can-edit-topic";
+    } else {
+      return "edit-topic";
+    }
+  }
+
+  @discourseComputed(
+    "site.mobileView",
+    "capabilities.touch",
+    "model.details.can_edit"
+  )
+  showEditButton(mobileView, touch, canEdit) {
```

Comment:
We're moving towards `mobileView` being purely viewport-width-based, so I don't think it's appropriate to use it here. Could we just use the `hover: none` media query?

Reply:
I made this change as part of my test-fixing, but lmk if you disagree and we can work it out (the change wasn't necessary to fix the tests)